### DISCUSSION TITLE
People Search - How can we return the same results sorting as we are getting on SharePoint online default search

### DISCUSSION THREAD

**Discussion by vishalgoyal2472** [microsoft-search/pnp-modern-search#4137] [General]
I have configured people search in PnP but its returned results are different and we asked this from Microsoft and they replied with below answer in quoted lines but I am not sure how I can configure same for PnP people search.

> As for the people result, when you asked about why one user was showing before, this is influenced by several factors to ensure that the most relevant results are presented to the user. Here are some key aspects that affect the ranking:
> 
> •	Relevance Signals: The search system uses various signals to determine the relevance of search results. These signals include the frequency of interactions, the recency of interactions, and the context of the search query. For example, if a user frequently interacts with "Lisa Gillmore," this interaction history can boost her ranking in the search results
> 
> •	Graph Signals: The Microsoft Graph plays a crucial role in enhancing search results by incorporating signals from various sources such as emails, calendar events, and documents. These signals help in creating a comprehensive understanding of the user's interactions and preferences, which in turn influences the ranking of search results
> 
> •	Personalization: The search experience is personalized based on the user's interactions and preferences. This means that the search results are tailored to the individual user, taking into account their unique context and history. For example, if a user has recently communicated with "Lisa Gillmore," she is more likely to appear higher in the search results
>

**Comment by kasperbolarsen** [comment]
The answer says it all. The out of the box people search is based on the base Microsoft People Search AND some additional pixie dust, so it is not possible for PnP Modern Search to get parity.
